### PR TITLE
Add high-level documentation of the methods of customization

### PR DESCRIPTION
**Pull request by OddBloke**

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -3,18 +3,29 @@
...
    contain the root `toctree` directive.
 
-Welcome to cpc-image-customization-examples's documentation!
-============================================================
+CPC Image Customization Examples
+================================
+
+This documentation provides examples of the ways in which the base
+Ubuntu cloud images from http://cloud-images.ubuntu.com can be
+customized to suite the needs and requirements of cloud partners.
```
Comment (rcj4747) [inline source/index.rst]: s/suite/suit/g

**Review by rcj4747 — CHANGES_REQUESTED** [review]
Spell checking won't always save you ;)

File: source/index.rst (hunk 1 above)
Comment (OddBloke) [reply source/index.rst]: Fixed.

**Review by rcj4747 — APPROVED** [review]
+1

**Review by josvazg — APPROVED** [review]
LGTM!

**Pull request merged**